index.html

```diff
@@ -7,13 +7,88 @@
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcuticon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header>
+     <div class="nav-menu">
+          <img class="top-logo" src="img/karma-logo.svg" alt="karma-logo" />
+        <nav >
+         <ul class="nav-menu">
+            <li class="nav-list">
+                <a href="#" class="nav-item">Meet Karma</a>
+            </li>
+            <li class="nav-list">
+                <a href="#" class="nav-item">How it works</a>
+            </li>
+            <li class="nav-list">
+                <a href="store.html" class="nav-item">Store</a>
+            </li>
+            <li class="nav-list">
+                <a href="#" class="nav-item">Blog</a>
+            </li>
+            <li class="nav-list">
+                <a href="#" class="nav-item">Help</a>
+            </li>
+            <li class="nav-list">
+                <a href="#" class="nav-item">Login</a>
+            </li>
+         </ul>
+       </nav>
+     </div>
+     </header>
+    <main>
+        <section class="home"> 
+            <div class="home-content">
+                <h2>Introducing Karma</h2>
+                <h3>Bring WiFI with you, everywhere you go.</h3>
+                <button class="learn">Learn More</button>
+            </div>
+        </section>
 
+        <section class="about">
+            <h2 class="h2-everyone">Everyone needs a little Karma.</h2>
+                <div class="about-divs">
+                    <article class="middle-article">
+                        <img class="middle-logo" src="img/icon-devices.svg" alt="icon-devices" />
+                        <p>Internet for all devices</p>
+                    </article>
+                    <article class="middle-article">
+                        <img class="middle-logo" src="img\icon-coffee.svg" alt="icon-coffee" />
+                        <p>Boost your productivity</p>
+                    </article>
+                    <article class="middle-article">
+                        <img class="middle-logo" src="img\icon-refill.svg" alt="icon-refill" />
+                        <p>Pay as You go</p>
+                    </article>
+                </div>
+            </h2>
+        </section>
+        <selection class="homepage-feature">
+            <div class="feature-image">
+                <img   src="img\homepage-feature.png" alt="Homepage feature">
+            </div> 
+            <div class="feature-content">
+                <p class="feature-text"><span>&ldquo;</span>Wherever I am, I just don't worry about my connection anymore!<span>&rdquo;</span> </p>
+                <button class="get-karma">Get Karma today</button>
```

Comment:
The image link is broken for me. Also need a bit of space between the button and edge of the box but this section is nearly there!